src/webportal/src/app/job-submission/utils/constants.js

```diff
@@ -144,3 +144,16 @@ export const DOCKER_OPTIONS = [
   },
 ];
 export const DEFAULT_DOCKER_URI = 'openpai/tensorflow-py36-cu90';
+// For PAI runtime only
+export const PAI_PLUGIN = 'com.microsoft.pai.runtimeplugin';
```

Comment:
I'm not sure if task's ssh connection will be a feature that could only be accessed by web portal.
If not, I think we should add some documents about job protocol's extra field format and move some logics to rest server.